ci/cron/daily-compat.yml

```diff
@@ -357,7 +357,7 @@ jobs:
         canton_local_path=external/$(ls canton/lib/*.jar)
         cd $dir
 
-        bad_file=daml-script/test/src/test-utils/com/daml/lf/engine/script/test/CantonFixture.scala
+        bad_file=daml-lf/integration-test-lib/src/main/com/daml/lf/CantonFixture.scala
```

Comment:
what is this "bad_file" about ?

Reply:
It's bad in that it includes an explicit path to the Canton jar, including a version number. So when we update the Canton dependency we have to update that file too.

See specifically [these lines](https://github.com/digital-asset/daml/blob/main/daml-lf/integration-test-lib/src/main/com/daml/lf/CantonFixture.scala#L138-L140):
```scala
          val cantonPath = rlocation(
            "external/canton/lib/canton-open-source-2.7.0-SNAPSHOT.jar"
          )
```